cfme-performance/tests/workloads/test_provisioning.py

```diff
@@ -0,0 +1,99 @@
+"""Runs Provisioning Workload."""
+from utils.appliance import clean_appliance
+from utils.appliance import get_server_roles_workload_provisioning
+from utils.appliance import set_server_roles_workload_provisioning
+from utils.appliance import wait_for_miq_server_ready
+from utils.grafana import get_scenario_dashboard_url
+from utils.log import logger
+from utils.providers import add_providers
+from utils.providers import retire_provisioned_vms
+from utils.providers import provision_vm
+from utils.providers import get_template_guids
+from utils.smem_memory_monitor import SmemMemoryMonitor
+from utils.ssh import SSHClient
+from utils.workloads import get_provisioning_scenarios
+from itertools import cycle
+import time
+import pytest
+
+
+@pytest.mark.parametrize('scenario', get_provisioning_scenarios())
+def test_provisioning(request, scenario):
+    """Runs through provisioning scenarios using the REST API to
+    continously order a catalog item(s) and provision a VM for a specified period of time.
+    Memory Monitor creates graphs and summary at the end of each scenario."""
```

Comment:
Lets re-word this description of the workload to indicate we are simply using the provisioning_requests api to provision a vm rather than ordering a catalog item.